packages/bullet.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Bullet < Package
+  description '3D Collision Detection and Rigid Body Dynamics Library'
+  homepage 'http://www.bulletphysics.com/Bullet/'
+  version '2.89'
+  compatibility 'all'
+  source_url 'https://github.com/bulletphysics/bullet3/archive/2.89.tar.gz'
+  source_sha256 '621b36e91c0371933f3c2156db22c083383164881d2a6b84636759dc4cbb0bb8'
+
+  depends_on 'libglu'
+
+  def self.build
+    system "cmake . -DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX} -DINSTALL_LIBDIR=#{CREW_LIB_PREFIX} -DCMAKE_BUILD_TYPE=Release"
```

Comment:
The generally accepted way to build using cmake is to create a build directory and call cmake from there.  This might work but it can mix the source with the build content and create a potential for conflicts.